tests/PHPStan/Rules/Functions/data/array_map.php

```diff
@@ -0,0 +1,6 @@
+<?php declare(strict_types = 1);
+
+array_map(
+	fn(stdClass $in): string => '',
```

Comment:
Self note: tests need to run on php  7.1, so these need to use a normal closure instead of the short version